lab/scenarios/vts_scenario.py

```diff
@@ -14,13 +14,35 @@ def loop(self):
         self._cloud.create_key_pair()
         internal_nets = self._cloud.create_net_subnet(common_part_of_name='internal', class_a=10, how_many=self._n_nets, is_dhcp=False)
 
+        ports = []
         for i in range(1, self._n_instances):
             port_pids = self._cloud.create_ports(instance_name=str(i), on_nets=internal_nets, is_fixed_ip=True)
             instance_name = self._cloud.create_instance(name=str(i), flavor='m1.medium', image=image_name, on_ports=port_pids)
 
+            ports += port_pids
+
             # cloud.cmd('ping -c5 {fip}'.format(fip=fips[i]))
             # instances[i] = Server(ip=fips[i], username='root', lab=None, name='a')
 
+        vtc1 = self._cloud.mediator.lab().get_node('vtc1')
```

Comment:
their might be a delay between port-binding oper in OS and time when those ports will be visible in VTC
also, it's good to have it as a helper in Vts class- like: 
Vts.check_port_id(port_id)